services/motion/builtin/replanner.go

```diff
@@ -19,19 +21,23 @@ type replanFn func(context.Context, [][]referenceframe.Input, int) (bool, error)
 
 // replanner bundles everything needed to execute a function at a given interval and return.
 type replanner struct {
+	id           string
 	period       time.Duration
 	responseChan chan replanResponse
 
 	// needReplan is a function that returns a bool describing if a replan is needed, as well as an error
 	needReplan replanFn
+	logger     golog.Logger
 }
 
 // newReplanner is a constructor.
-func newReplanner(period time.Duration, fnToPoll replanFn) *replanner {
+func newReplanner(period time.Duration, fnToPoll replanFn, id string, logger golog.Logger) *replanner {
```

Comment:
this will be removed and is here to help me investigate testing behavior from the CI